content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -397,8 +397,10 @@ liveness and readiness checks:
 
 * `initialDelaySeconds`: Number of seconds after the container has started before startup,
   liveness or readiness probes are initiated. If a startup  probe is defined, liveness and
-  readiness probe delays do not begin until the startup probe has succeeded. Defaults to 0
-  seconds. Minimum value is 0.
+  readiness probe delays do not begin until the startup probe has succeeded.
+  `initialDelaySeconds` will block probes, but `periodSeconds` remains active. The probe may
```

Comment:
by "active", do you mean "effective"?

Reply:
Better be explicit when you say "the probe". Which probe are you referring to in this context?

Reply:
The original wording is much more clear for me. I can't think of a suggestion for rewording this, but I don't think the current phrasing is an improvement.

Maybe something like:

```txt
An `initialDelaySeconds` value greater than 0 will delay when the first probe starts. If `periodSeconds` is less than the `initialDelaySeconds` regular periodic probes will begin. For example, if `initialDelaySeconds` is set to 20, but `periodSeconds` is set to 10, periodic probes will still begin after an initial 10 seconds.
```

Reply:
Thanks for the suggestions. This is probably still a bit confusing, because I was trying to say if `initialDelaySeconds` is set to 20, `periodSeconds` is set to 10, the initial probe is around 30; if `initialDelaySeconds` is set to 10, `periodSeconds` is set to 20, the initial probe is around 20.

Reply:
I have combined your suggestions into a modified explanation, does that sound clearer? Thanks.